### ISSUE TITLE
[BUG] TransactionPackets caused NoiseDistraction

### ISSUE THREAD

**Issue by C0an** [VerusAC/VerusAntiCheat#14]
Hi, for some reason whenever the TransactionPacket is sent it makes my Microwave receit the composition of Mozart's Symphony No. 40 in G Minor, K. 550: I. Molto allegro in the style of a Nokia 3310 Ringtone from an Arabic Country, please resolve this bug ASAP as I wish to continue to use my Microwave without further issue as it could deem as a potential safety hazard.

**Comment by OmenDoesStuff** [comment]
no.

**Comment by AkramLZ** [comment]
> Hi, for some reason whenever the TransactionPacket is sent it makes my Microwave receit the composition of Mozart's Symphony No. 40 in G Minor, K. 550: I. Molto allegro in the style of a Nokia 3310 Ringtone from an Arabic Country, please resolve this bug ASAP as I wish to continue to use my Microwave without further issue as it could deem as a potential safety hazard.

Don't worry, it will be solved within the next 7-9 months

**Comment by OmenDoesStuff** [comment]
> > Hi, for some reason whenever the TransactionPacket is sent it makes my Microwave receit the composition of Mozart's Symphony No. 40 in G Minor, K. 550: I. Molto allegro in the style of a Nokia 3310 Ringtone from an Arabic Country, please resolve this bug ASAP as I wish to continue to use my Microwave without further issue as it could deem as a potential safety hazard.
> 
> Don't worry, it will be solved within the next 7-9 months

wrong, jacob is too lazy to fix it. thats why verus is open source now.
